fury/material.py

```diff
@@ -2,43 +2,86 @@
+    metallic : float, optional
         Metallic or non-metallic (dielectric) shading computation value. Values
         must be between 0.0 and 1.0.
     roughness : float, optional
         Parameter used to specify how glossy the actor should be. Values must
         be between 0.0 and 1.0.
+    anisotropy : float, optional
+        Isotropic or anisotropic material parameter. Values must be between
+        0.0 and 1.0.
+    anisotropy_rotation : float, optional
+        Rotation of the anisotropy around the normal in a counter-clockwise
+        fashion. Values must be between 0.0 and 1.0. A value of 1.0 means a
+        rotation of 2 * pi.
+    coat_strength : float, optional
+        Strength of the coat layer. Values must be between 0.0 and 1.0 (0.0
+        means no clear coat will be modeled).
+    coat_roughness : float, optional
+        Roughness of the coat layer. Values must be between 0.0 and 1.0.
+    base_ior : float, optional
+        Index of refraction of the base material. Default is 1.5. Values must
+        be between 1.0 and 2.3.
+    coat_ior : float, optional
+        Index of refraction of the coat material. Default is 1.5. Values must
+        be between 1.0 and 2.3.
 
     """
     if not VTK_9_PLUS:
-        warnings.warn('Your PBR effect cannot be apply due to VTK version. '
+        warnings.warn('Your PBR effect cannot be applied due to VTK version. '
                       'Please upgrade your VTK version (should be >= 9.0.0).')
-        return
+        return None
 
     try:
         prop = actor.GetProperty()
         try:
             prop.SetInterpolationToPBR()
-            prop.SetMetallic(metallicity)
-            prop.SetRoughness(roughness)
+
+            pbr_params = {'metallic': metallic, 'roughness': roughness}
+
+            prop.SetMetallic(pbr_params['metallic'])
+            prop.SetRoughness(pbr_params['roughness'])
+            if VTK_9_1_PLUS:
+                pbr_params['anisotropy'] = anisotropy
+                pbr_params['anisotropy_rotation'] = anisotropy_rotation
+                pbr_params['coat_strength'] = coat_strength
+                pbr_params['coat_roughness'] = coat_roughness
+                pbr_params['base_ior'] = base_ior
+                pbr_params['coat_ior'] = coat_ior
+
+                prop.SetAnisotropy(pbr_params['anisotropy'])
+                prop.SetAnisotropyRotation(pbr_params['anisotropy_rotation'])
+                prop.SetCoatStrength(pbr_params['coat_strength'])
+                prop.SetCoatRoughness(pbr_params['coat_roughness'])
+                prop.SetBaseIOR(pbr_params['base_ior'])
+                prop.SetCoatIOR(pbr_params['coat_ior'])
+            else:
+                warnings.warn(
+                    'Anisotropy and Clear coat based PBR effects cannot be '
+                    'applied due to VTK version. Please upgrade your VTK '
+                    'version (should be >= 9.1.0).')
+            return pbr_params
```

Comment:
we do not need this `if` assuming we depend only on 9.1

Reply:
Agree. I will assume our min version is 9.1 and adjust accordingly.

Reply:
Changed.